src/GameSrc/olh.c

```diff
@@ -488,6 +488,7 @@ void olh_overlay(void) {
     uiPopGlobalCursor();
     uiFlush();
     olh_overlay_on = FALSE;
+	gr_clear(0); //makes red pixels go away, but real problem is probably in REF_IMG_bmBlankMFD
```

Comment:
It looks like we're using spaces for indentation and you're using tabs, not a big deal but it would be nice to be consistent.